app/scripts/modules/amazon/src/instance/details/instance.details.controller.js

```diff
@@ -118,6 +132,23 @@ module.exports = angular.module('spinnaker.amazon.instance.details.controller',
                     return true;
                   }
                 });
+              }) ||
```

Comment:
seeing the `||` makes me think maybe these matching functions should be split up to make this a little more readable.

(I know it's already messy code, but this is...um...)